### ISSUE TITLE
Timeline Vertical Scrollbar - scrollbar width not recalculated on browser zoom

### ISSUE THREAD

**Issue by garrettloy** [almende/vis#2333]
Hi!

I have a straightforward issue with a proposed fix.

Zooming in or out in the browser with verticalScroll:true on the timeline does not recalculate the width of the scrollbar, which throws off the width of group labels:

![image](https://cloud.githubusercontent.com/assets/7387847/20441209/7118868c-ad78-11e6-8d33-6e3eec78c375.png)

proposed fix: add the starred line to Core.prototype._startAutoResize
```

  Core.prototype._startAutoResize = function () {
    var me = this;
  
    this._stopAutoResize();

    this._onResize = function () {
      if (me.options.autoResize != true) {
        // stop watching when the option autoResize is changed to false
        me._stopAutoResize();
        return;
      }
  
      if (me.dom.root) {
        // check whether the frame is resized
        // Note: we compare offsetWidth here, not clientWidth. For some reason,
        // IE does not restore the clientWidth from 0 to the actual width after
        // changing the timeline's container display style from none to visible
        if (me.dom.root.offsetWidth != me.props.lastWidth || me.dom.root.offsetHeight != me.props.lastHeight) {
          me.props.lastWidth = me.dom.root.offsetWidth;
          me.props.lastHeight = me.dom.root.offsetHeight;
          **me.props.scrollbarWidth = util.getScrollBarWidth();**
  
          me.body.emitter.emit('_change');
        }
      }
    };
```

Thanks!

**Comment by yotamberk** [comment]
Thanks @garrettloy (both for reporting and for the solution suggestion!)
Do you want to submit a PR with this fix?

**Comment by mojoaxel** [comment]
Fixed by @garrettloy with #2344
